domain/lib/src/model/sort/sorter.dart

```diff
@@ -65,7 +65,7 @@ extension SorterExtension on Sorter {
     return <Sorter>[
       Sorter(orderScreen, OrderBy.modificationDate, OrderType.descending),
       Sorter(orderScreen, OrderBy.creationDate, OrderType.descending),
-      Sorter(orderScreen, OrderBy.fileSize, OrderType.descending),
+      if (orderScreen != OrderScreen.sharedSpaceWorkGroup) Sorter(orderScreen, OrderBy.fileSize, OrderType.descending),
```

Comment:
I thought sorting by file size was in every view ? (@hoangdat can you confirm pls)

Reply:
Hmm. We dev features base on Web platform currently. I don't see a column sorter for the file size

Reply:
You're right, maybe it's because we can't get folder size

Reply:
please make it consistent with web version, and follow the story